src/alchemlyb/estimators/mbar_.py

```diff
@@ -67,13 +73,15 @@ def __init__(
         relative_tolerance=1.0e-7,
         initial_f_k=None,
         method="robust",
+        n_bootstraps=0,
```

Comment:
I'd add a reminder comment
```python
 n_bootstraps=0,    # release 2.2: change to 50 (see PR #322)
```

Reply:
So my feeling is that for the MBAR estimator, we retain the current behaviour unless people manually turn it on as bootstrap does come with a computational cost.

It is only for the estimate method in workflow.ABFE where I know that the input is decorrelated and I'm prepared to pay the computational cost, I will set the default to 50.

Reply:
Yes, that's ok, we just can't make the default behave differently without a minimal warning period.